journal-of-contemporary-water-research-and-education.csl

```diff
@@ -0,0 +1,625 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="display-and-sort" page-range-format="chicago">
+  <info>
+    <title>Journal of Contemporary Water Research and Education</title>
+    <title-short>JCWRE</title-short>
+    <id>http://www.zotero.org/styles/journal-of-contemporary-water-research-and-education</id>
+    <link href="http://www.zotero.org/styles/journal-of-contemporary-water-research-and-education" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-author-date" rel="template"/>
+    <link href="https://ucowr.org/wp-content/uploads/2020/11/Original-Research-Article-Author-Instructions.pdf" rel="documentation"/>
+    <author>
+      <name>Michael Schramm</name>
+      <email>mpschramm@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <issn>1936-704X</issn>
+    <summary>The  based of the author-date variant of the Chicago style</summary>
```

Comment:
Looks like this is missing something. You don't have to refer to the Chicago style here. That's covered by the template link above.